client/components/mma/cancel/cancellationSaves/supporterplus/SupporterPlusOffer.tsx

```diff
@@ -0,0 +1,302 @@
+			position: absolute;
+			top: ${space[6] + space[4]}px;
+		}
+	`;
+
+	const strikethroughPriceCss = css`
+		${textSans20};
+		color: ${neutral[46]};
+		margin: 0;
+	`;
+
+	const offerBoxTitleCss = css`
+		${textSansBold28};
+		color: ${neutral[7]};
+		margin: 0;
+	`;
+
+	const billingResumptionDateCss = css`
+		${textSans15};
+		color: ${neutral[38]};
+		margin: 0;
+	`;
+
+	const offerButtonCss = css`
+		margin: ${space[5]}px 0 ${space[6]}px;
+		width: 100%;
+		justify-content: center;
+	`;
+
+	const cancelBtnHolderCss = css`
+		${from.desktop} {
+			display: flex;
+			justify-content: space-between;
+		}
+	`;
+
+	const cancelButtonCss = css`
+		margin: 0 0 ${space[3]}px;
+		width: 100%;
+		justify-content: center;
+		${from.desktop} {
+			width: fit-content;
+		}
+	`;
+
+	const termsCss = css`
+		${textSans12};
+		color: ${palette.neutral[46]};
+		margin-top: ${space[3]}px;
+	`;
+
+	return (
+		<>
+			<ProgressStepper
+				steps={[{}, {}, { isCurrentStep: true }, {}]}
+				additionalCSS={css`
+					margin: ${space[8]}px 0 ${space[9]}px;
+				`}
+			/>
+			<Heading
+				borderless
+				cssOverrides={[
+					measure.heading,
+					css`
+						margin-bottom: ${space[2]}px;
+					`,
+				]}
+			>
+				This doesn't have to be goodbye
+			</Heading>
+			<h3 css={standfirstCss}>
+				Instead of cancelling, enjoy two months with all your existing
+				benefits — for free.
+			</h3>
+			<div css={availableOfferBoxCss}>
+				<picture css={mobileHeaderImageCss}>
+					<source
+						srcSet="https://media.guim.co.uk/c9287b29f36749b27a6d85a8fa07d865aa7c7299/0_0_1660_1868/889.png"
+						media="(min-width: 980px)"
+					/>
+					<img src="https://media.guim.co.uk/63d17ee19313703129fbbeacceaafcd6d1cc1014/0_0_1404_716/500.png" />
+				</picture>
+				<Ribbon
+					copy="Your one-time offer"
+					ribbonColour={palette.brand[500]}
+					copyColour={palette.neutral[100]}
+					roundedCornersRight
+					withoutTail
+					small
+					additionalCss={ribbonCss}
+				/>
+				<div css={availableOfferBoxInnerCss}>
+					{isPaidSubscriptionPlan(mainPlan) && (
+						<p css={strikethroughPriceCss}>
+							<s>
+								{mainPlan.currency}
+								{mainPlan.price / 100}/{mainPlan.billingPeriod}
+							</s>
+						</p>
+					)}
+
+					<h4 css={offerBoxTitleCss}>
+						{capitalize(offerPeriodWord)} {offerPeriodType} free
+					</h4>
+					<p css={billingResumptionDateCss}>
+						Billing resumes on {nextNonDiscountedPaymentDate}
+					</p>
+					<Button
+						onClick={() => {
+							navigate('../offer-review', {
+								state: routerState,
+							});
+						}}
+						cssOverrides={offerButtonCss}
+					>
+						Redeem your offer
+					</Button>
+					<p
+						css={css`
+							margin: 0 0 ${space[5]}px;
+							${textSans15};
+						`}
+					>
+						Keep your existing benefits:
+					</p>
+					<div
+						css={css`
+							max-width: 290px;
+						`}
+					>
+						<BenefitsSection
+							small
+							benefits={[
+								{
+									name: '',
```

Comment:
I wonder if the `name` attribute should be made optional (instead of having to pass empty strings)?